gunicorn.conf.py

```diff
@@ -16,22 +16,6 @@
 if 'H_GUNICORN_KEYFILE' in os.environ:
     keyfile = os.environ['H_GUNICORN_KEYFILE']
 
-if not os.environ.get('GUNICORN_STATS_DISABLE', None):
```

Comment:
I've completely removed the `GUNICORN_STATS_DISABLE` envvar. I don't know why we ever needed this envvar. It's not used in dev, qa or prod. It came from this PR: https://github.com/hypothesis/h/pull/3067

Anyway, since we're completely removing everything that `GUNICORN_STATS_DISABLE` disabled anyway, we don't need it anymore